### ISSUE TITLE
fails to parse download option --output - unrecognized arguments

### ISSUE THREAD

**Issue by jtmoon79** [ihabunek/twitch-dl#86]
[According to the README.md](https://github.com/ihabunek/twitch-dl/blob/4ff4c8763c79ce5a7468912fe9883b4592397652/README.md)

> Overriding file name
> The target filename can be defined by passing the --output option followed by the desired file name,
> The filename uses Python format string syntax and may contain placeholders in curly braces which will be replaced with relevant information tied to the downloaded video.

I attempted to use download option `--output` and it failed
```
$ python3 -m pipx run -- twitch-dl download -q source --output "{date}_{id}_{channel_login}_{title_slug}.{format}" '1255522958'
usage: twitch-dl [-h] [--version] {videos,clips,download,info} ...
twitch-dl: error: unrecognized arguments: --output 1255522958
```
If not passed `--output "{date}_{id}_{channel_login}_{title_slug}.{format}"` then the command is parsed successfully and the download begins.

Using
```
$ python3 --version
Python 3.9.5

$ python3 -m pipx --version
1.0.0

$ python3 -m pipx run -- twitch-dl --version
twitch-dl v1.17.1
```
on Ubuntu 20.

**Comment by ihabunek** [comment]
Yeah, I pushed that feature yesterday and haven't made a release yet.

**Comment by ihabunek** [comment]
Released in [1.18.0](https://github.com/ihabunek/twitch-dl/releases/tag/1.18.0).
